lib/absinthe/type/field.ex

```diff
@@ -111,6 +121,47 @@ defmodule Absinthe.Type.Field do
      for the field (and useful for complex resolutions using the resolved source
      object, etc)
 
+  ## Complexity function
+
+  ### Default
+
+  If no complexity function is given, the default complexity function is used,
+  which is equivalent to:
+
+      fn(_, child_complexity) -> 1 + child_complexity end
+
+  ### Custom Complexity
+
+  When accepting arguments, however, you probably need do use them for
+  something. Here's an example of definining a field that looks up at most
+  `limit` users:
+  ```
+  query do
+    field :users, :person do
+      arg :limit, :integer
+
+      complexity fn %{limit: limit}, child_complexity ->
+        10 + limit * child_complexity
+      end
+    end
+  end
+  ```
+
+  Custom complexity functions are passed two arguments:
```

Comment:
Update this documentation to also include the 3-arity version of the complexity function?